ush/cam/evs_href_grid2obs_product.sh

```diff
@@ -27,16 +29,19 @@ for prod in mean prob ; do
 
 
     for valid_run in run1 run2 run3 run4 ; do
-
-     
-     #***********************
      # Build sub-jobs
      # **********************
      >run_href_${model}.${dom}.${valid_run}_product.sh
+     ######################################################################################################
+     #Restart: check if this CONUS task has been completed in the precious run
```

Comment:
Hi Binbin! Can you correct the typo in line 36 of `ush/cam/evs_href_grid2obs_product.sh`? Correct "precious" to be "previous". Thanks!

Reply:
Actually, I am seeing the "precious" typo in multiple files. Please update the following:
```
ush/cam/evs_href_spcoutlook.sh
ush/cam/evs_href_snowfall.sh
ush/cam/evs_href_grid2obs_system.sh
ush/cam/evs_href_grid2obs_profile.sh
ush/cam/evs_href_grid2obs_product.sh
ush/cam/evs_href_precip.sh
```